src/docs/release/breaking-changes/template.md

```diff
@@ -68,44 +116,53 @@ Code after migration:
 
 ## Timeline
 
-Landed in version: xxx<br>
-In stable release: not yet
-
 {% comment %}
-The version # of the SDK where this change was
-introduced.  If there is a deprecation window,
-the version # to which we guarantee to maintain the old API.
+  The version # of the SDK where this change was
+  introduced.  If there is a deprecation window,
+  the version # to which we guarantee to maintain
+  the old API. Use the following template:
 {% endcomment %}
 
+Landed in version: xxx<br>
+In stable release: not yet
+
 ## References
 
 {% comment %}
-These non-functional links are commented out because they are messing up the
-Travis build. Remove the comment tags once you fill this in with real links.
+  These non-functional links are commented out because
+  they are messing up the Travis build.
+  Remove the comment tags once you fill this in with
+  real links. Only use the "master-api" include if
+  you link to "master-api.flutter.dev".
 
 {% include master-api.md %}
 
 API documentation:
+
 * [`ClassName`][]
 
 Relevant issues:
+
 * [Issue xxxx][]
 * [Issue yyyy][]
 
 Relevant PRs:
+
 * [PR title #1][]
 * [PR title #2][]
 {% endcomment %}
 
 {% comment %}
-Add the links to the end of the file in alphabetical order.
-These faux links are commented out because they make
-the Travis build believe they are broken links,
-but please remove the comment lines before you commit!
-
-If you need to link to the docs on the master channel,
-please include the following note and make sure that
-the URL includes the master link (as shown below).
+  Add the links to the end of the file in alphabetical order.
+  These faux links are commented out because they make
+  the Travis build believe they are broken links,
+  but please remove the comment tags before you commit!
```

Comment:
```suggestion
  The following faux links are commented out because they
  break the Travis build.
  Please remove the comment tags before you commit!
```